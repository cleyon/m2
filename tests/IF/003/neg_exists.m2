@if !exists(non-existing-file)
GOOD Non-Existing file okay
@else
BAD Non-Existing file failed
@fi
@if !exists(existing-file)
BAD Failed to detect existing file
@else
GOOD Existent file okay
@fi
