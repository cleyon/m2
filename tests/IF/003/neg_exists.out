GOOD Non-Existing file okay
GOOD Existent file okay
